.github/workflows/build_wheels.yml

```diff
@@ -20,9 +24,10 @@ jobs:
   build_wheels:
     name: Build wheel for ${{ matrix.os }}, Python ${{ matrix.pyver }}
     runs-on: ${{ matrix.os }}
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'build_wheels') }}
```

Comment:
The tag builds will not match this condition. You will need to add an "or" to this condition that matches tags.

Reply:
Will this job still run automatically on pushes to master/next that are not tagged? If not, I will need a condition for that as well.

Reply:
No. `github.event.pull_request.labels.*.name` only exists in pull requests so the condition `contains(github.event.pull_request.labels.*.name, 'build_wheels')` will be false on both tag triggered and push triggered builds.

Reply:
I think I've updated the conditions correctly.